src/server/heat_automation.py

```diff
@@ -189,7 +189,7 @@ def run_auto_frequency(self, heat_or_id, current_frequencies, num_nodes, calc_fn
                         for node in available_seats:
                             end_idx = len(used_frequencies) - 1
                             for f_idx, pilot_freq in enumerate(used_frequencies):
-                                if node['frq']['f'] == pilot_freq['f']:
+                                if node['frq']['f'] == pilot_freq['f'] and node['frq']['b'] == pilot_freq['b']:
```

Comment:
freqs are identical on some channels (like R7 and D6) and the current system was selecting R band for DJI pilots, which is not what we want.